mne/viz/evoked.py

```diff
@@ -1822,6 +1841,22 @@ def _get_data_and_ci(evoked, scaling=1, picks=None, ci_fun=None, gfp=False):
         return data,
 
 
+def _calculate_ci_and_mean(evokeds, conditions, scaling, picks, ci_fun, gfp):
+    """Calculate time series and CI, potentially aggregating over sensors."""
+    ci_dict, data_dict = dict(), dict()
+
+    for cond in conditions:
+        this_evokeds = evokeds[cond]
+        # this will fail if evokeds do not have the same structure
+        # (e.g. channel count)
+        res = _get_data_and_ci(this_evokeds, scaling=scaling, picks=picks,
```

Comment:
can you write

`data, ci = _get_data_and_ci(...)`

it makes it explicit what is returned.

Reply:
then `_get_data_and_ci` would have to return `data, None` when no ci is requested, currently it returns `(data, )`